plugins/rollbar_dashboards/app/models/rollbar_dashboards/setting.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+module RollbarDashboards
+  class Setting < ActiveRecord::Base
+    self.table_name = 'rollbar_dashboards_settings'
+
+    belongs_to :project
+
+    validates :base_url, :read_token, :time_zone, presence: true
+
+    def resolved_read_token
```

Comment:
if this is used on every project show/deploy-show then we might want to cache that ... also not blow up and make the project undeployable in cas vault has a hickup

Reply:
If someone were to update the token and the token was cached then the feature would be broken until the cached value expired? I could see that leading to some confusion. I could cache the result to the `RollbarDashboards::Client` call on the deploy page as well? Then the entire dashboard is essentially cached for 2 minutes, including the token.